buildscripts/build-native.sh

```diff
@@ -40,7 +45,14 @@ build_native_corert()
 
     # Regenerate the CMake solution
     echo "Invoking cmake with arguments: \"$__ProjectRoot\" $__BuildType"
-    "$__ProjectRoot/src/Native/gen-buildsys-clang.sh" "$__ProjectRoot" $__ClangMajorVersion $__ClangMinorVersion $__BuildArch $__BuildType
+    if [ $__BuildArch == "wasm" ]; then
+        # TODO: Add a real wasm build
+        echo "Wasm build is not currently implemented"
```

Comment:
Have you attempted running emcmake to see if it just works with the rest of this build? (No need to block on this, just curious if it should work already)

Reply:
At the moment, I am building with:

```
emcmake cmake \
    "-DCMAKE_TOOLCHAIN_FILE=$EMSCRIPTEN/cmake/Modules/Platform/Emscripten.cmake" \
    "-DCMAKE_BUILD_TYPE=$build_type" \
    "$1/src/Native"
```

Unfortunately the rest of the build doesn't just work, but with a few tweaks to the `CMakeLists.txt` files, `.\Native\jitinterface` and `.\Native\Bootstrap` compile. I am currently running into issues with `.\Native\Runtime\Portable` with the "PalRedHawk" files, as the new target is not handled there. The assembly files for `.\Native\Runtime\Full` are also going to be an issue. 

If you like, I could add what I have as a work in progress MR so you could see where it is at.